ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -60,17 +60,11 @@ platform:
           username: {{ hostvars[host]['ipmi_user'] }}
           password: {{ hostvars[host]['ipmi_password'] }}
         bootMACAddress: {{ hostvars[host]['provision_mac'] }}
-{% if ((release_version[0]|int == 4) and (release_version[2]|int < 5)) %}
```

Comment:
The goal should be not to remove the root device hints but instead just shift from 4.5 to 4.6 so changing this line 
to:
```
{% if ((release_version[0]|int == 4) and (release_version[2]|int < 6)) %}
```